CHANGELOG.md

```diff
@@ -0,0 +1,182 @@
+# Changelog
+
+This changelog consolidates changes for the [nilrt](https://github.com/ni/nilrt) repo, [meta-nilrt](https://github.com/ni/meta-nilrt) OE layer, and NI's branch of the [linux](https://github.com/ni/linux) kernel. Other OE layer changes are tracked in their respective changelogs.
+
+The NI Linux Real-Time project uses a mainline branching model with product branches for stable releases. The distro itself is versioned according to a yearly release cadence and *is not* semantically versioned.
+
+## [Unreleased]
+Branch: `nilrt/master/sumo`
```

Comment:
Should this say dunfell not sumo?

Reply:
My thoughts were: since we have and will likely continue to have two release mainlines running in parallel, that this branch `master/sumo` would have a CHANGELOG file with the `[Unreleased]` branch being sumo, and the `master/dunfell` ref would have the same, but for dunfell.

When one of the branches goes stable, we can cherry-pick the changelog for that stable version over to the other branch.

What do you think? Is there an easier way to track those changes in parallel?

Reply:
Makes sense. Should have payed attention which branch this is going into. Disregard my comment.